sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -106,10 +109,12 @@ def _setup_tracing(configurations: Dict[str, ConfigurationValue]):
     )
     set_tracer_provider(tracer_provider)
     trace_exporter = AzureMonitorTraceExporter(**configurations)
-    span_processor = BatchSpanProcessor(
+    bsp = BatchSpanProcessor(
         trace_exporter,
     )
-    get_tracer_provider().add_span_processor(span_processor) # type: ignore
+    for span_processor in configurations[SPAN_PROCESSORS_ARG]: # type: ignore
+        get_tracer_provider().add_span_processor(span_processor)
```

Comment:
nit: Let's rearrange to add exporter and bsp code after we add the other processors to make it less confusing.